src/types/adapter.ts

```diff
@@ -164,6 +164,11 @@ export type ProtocolDetails = {
   assetDetails: AssetDetails
 }
 
+export interface GetPositionsInputWithMandatoryTokenAddresses
```

Comment:
nit: `GetPositionsInputWithTokenAddresses` just to shorten it a bit whilst keeping the meaning.